routes/api/member.js

```diff
@@ -92,6 +95,44 @@ router.route("/update-many").put(function(req, res) {
     .catch(err => {
       res.status(500).send(err);
     });
+
+  });
+    
+// resolves as /api/member/:id    
+router
+  .route("/:id")
```

Comment:
Same concern as in the previous api route document. Especially because we're adding these routes to an object, which is not ordered, I don't have faith that `/:id` won't get triggered by `/update-many`, for example, and just pass in "update-man" as the id. If that did happen, of course that could get pretty messy. Have you done any testing of the old routes after making this addition?